CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to Azure DevOps Extension
+
+Welcome, and thank you for your interest in contributing to Azure DevOps Extension!
+
+There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
+
+# Have a question
+
+Search existing github issues for similar questions and feel free to file an issue with a "question" tag to get our attention. 
+
+# Found a Bug
+
+#### Where to Find Known Issues
+
+Before you file a new bug report do check on existing open [issues](https://github.com/Microsoft/vsts-cli/issues?q=is%3Aopen+is%3Aissue+label%3Abug).
+
+If you find your issue already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment.
+👍 - upvote 👎 - downvote
```

Comment:
looks overkill 😄

Reply:
This section needs to be retained. We really have to consider the persona of users who are entering the repo with absolutely no context and this is crucial.